packages/cli/src/UserManagement/routes/users.ts

```diff
@@ -478,6 +490,8 @@ export function usersNamespace(this: N8nApp): void {
 
 			void InternalHooksManager.getInstance().onUserDeletion(req.user.id, telemetryData, false);
 
+			await this.externalHooks.run('user.deleted', [userToDelete.email]);
```

Comment:
also here I think the sanitized `userToDelete` provides more flexibility

Reply:
done